lib/esbonio/hatch.toml

```diff
@@ -14,12 +14,17 @@ extra-dependencies = ["pytest-lsp>=0.3.1,<1"]
 matrix-name-format = "{variable}{value}"
 
 [[envs.hatch-test.matrix]]
-python = ["3.9", "3.10", "3.11", "3.12"]
+python = ["3.9", "3.10", "3.11", "3.12", "3.13"]
 
 [[envs.hatch-test.matrix]]
 python = ["3.9", "3.10", "3.11", "3.12"]
 sphinx = ["5", "6", "7"]
 
+[[envs.hatch-test.matrix]]
+python = ["3.13"]
+sphinx = ["6", "7"]
```

Comment:
Sphinx v5 fails with the following error ` Could not import extension sphinx.builders.epub3 (exception: No module named 'imghdr')`

[`imghdr`](https://docs.python.org/3.12/library/imghdr.html) was removed in Python 3.13